lib/src/indicators/audio_equalizer.dart

```diff
@@ -58,7 +58,7 @@ class _AudioEqualizerState extends State<AudioEqualizer>
 
   @override
   Widget build(BuildContext context) {
-    final widgets = List<Widget>(7);
+    final widgets = List<Widget>.filled(7, Container());
```

Comment:
```suggestion
    final empty = Placeholder()
    final widgets = List<Widget>.filled(7, empty);
```

Reply:
I think it's better to use SizedBox() because it is more resource-efficient.

Reply:
```suggestion
    final empty = SizedBox();
    final widgets = List<Widget>.filled(7, empty);
```